### PR TITLE
Added flag for operator only features

### LINKED ISSUE #143: Create operator only flag for mcs
Create an operator flag for mcs, it is required that this flag displays the correspondent menu in mcs

### PR DESCRIPTION
**Pull request by bexsoft**
fixes #143 

## What does this do?
Added flag to only enable operator endpoints / links in mcs

## How to test

- Enable MCS_OPERATOR_ONLY flag as "on" before running server
-  Run server as usual

Screen Shot 2020-05-26 at 3 26 04 PM
Screen Shot 2020-05-26 at 3 25 06 PM

### CODE REVIEW

**Comment by cesnietor** [comment]
is not working on port 9090:
Screen Shot 2020-05-26 at 2 33 30 PM

```
export MCS_OPERATOR_ONLY=on && export MCS_ACCESS_KEY=minio &&
export MCS_SECRET_KEY=minio123 && MCS_MINIO_SERVER=http://localhost:9000 ./mcs server
2020/05/26 14:30:24 Serving mcs at http://127.0.0.1:9090
2020/05/26 14:31:42 Redirecting /static/css/2.e681e74d.chunk.css to index.html.
2020/05/26 14:31:42 Redirecting /static/js/2.6205f397.chunk.js to index.html.
2020/05/26 14:31:42 Redirecting /static/js/main.579bbb03.chunk.js to index.html.
2020/05/26 14:31:45 Redirecting /static/css/2.e681e74d.chunk.css to index.html.
2020/05/26 14:31:45 Redirecting /static/js/2.6205f397.chunk.js to index.html.
2020/05/26 14:31:45 Redirecting /static/js/main.579bbb03.chunk.js to index.html.
2020/05/26 14:31:58 Redirecting /static/css/2.e681e74d.chunk.css to index.html.
2020/05/26 14:32:40 Redirecting /static/css/2.e681e74d.chunk.css to index.html.

```

**Comment by cesnietor** [comment]
also the assets were not built

**Comment by bexsoft** [comment]
> Remove this: Tes

I couldn't find this test :/

**Comment by cesnietor** [comment]
> > Remove this: Tes
> 
> I couldn't find this test :/

it is in ListClusters.tsx.
```
<Grid item xs={12}>
          REMOVE THIS:: <Link to={"/clusters/demoCluster"}>Test</Link>
          <br />
        </Grid>
```

**Comment by bexsoft** [comment]
> > > Remove this: Tes
> > 
> > 
> > I couldn't find this test :/
> 
> it is in ListClusters.tsx.
> 
> ```
> <Grid item xs={12}>
>           REMOVE THIS:: <Link to={"/clusters/demoCluster"}>Test</Link>
>           <br />
>         </Grid>
> ```

Removed :)

File: restapi/user_session.go [hunk 1]
```diff
@@ -53,6 +53,9 @@ func getSessionResponse(sessionID string) (*models.SessionResponse, error) {
 		log.Println("error getting claims from JWT", err)
 		return nil, errorGenericInvalidSession
 	}
+
+	acl.OperatorOnly = OperatorOnly
```
Comment (dvaldivia) [inline restapi/user_session.go:57]: acl.OperatorOnly = GetOperatorOnly() would be enough here

Comment (bexsoft) [reply restapi/user_session.go]: Removed operatorOnly from restapi & loaded from acl package

**Pull request merged**